src/tmx/TmxCtl/src/lib/TmxControl.cpp

```diff
@@ -162,20 +162,22 @@ bool TmxControl::checkPerm()
 	if (isRoot)
 		return true;
 
-	struct group *admGrp = ::getgrnam("adm");
-
-	if (!admGrp)
+	struct group admGrp;
```

Comment:
There is a sonar scanner reported bug about this variable being uninitialized ( a garbage value)

Reply:
Paul, I noted this in my first comment on this PR.  It appears to be a false positive to me because the struct is filled in during the getgrnam_r call or the function returns if it does not work.  Let me know what you think.